pkg/config/config.go

```diff
@@ -41,22 +41,3 @@ func NewDefaultConfig(fs afero.Fs) *Config {
 		Fs:                   fs,
 	}
 }
-
-// NewConfig returns a new Config for a manager of controllers
```

Comment:
:thinking: I guess that will break tests now as https://github.com/cloudfoundry-incubator/quarks-utils/pull/42 added a couple to cover it.. sorry :smiley:

Reply:
I saw that, however this function appears to be unused ...

Reply:
Looks good indeed! I've checked that too when adding the test.. but then wasn't sure if was being consumed elsewhere